go/datas/database_common.go

```diff
@@ -188,12 +193,15 @@ func (dbc *databaseCommon) getRootAndDatasets() (currentRootHash hash.Hash, curr
 	return
 }
 
-func (dbc *databaseCommon) tryUpdateRoot(currentDatasets types.Map, currentRootHash hash.Hash) (err error) {
-	// TODO: This Map will be orphaned if the UpdateRoot below fails
+func (dbc *databaseCommon) tryCommitChunks(currentDatasets types.Map, currentRootHash hash.Hash) (err error) {
```

Comment:
There's already a method on databaseCommon called `doCommit()`, and to me `tryCommit` implies "I'll attempt a doCommit and return an error instead of panicing if bad things happen" or some such. This name is an attempt to indicate that this is trying an operation on the ChunkStore.